sdk/firstload/src/__tests__/setup.ts

```diff
@@ -1,4 +1,5 @@
 import { vi } from 'vitest'
+import 'vitest-canvas-mock'
```

Comment:
Unrelated, but fixes some errors we were seeing in the console when running tests.